app/forms/decidim/census_connector/verifications/census/verification_handler.rb

```diff
@@ -11,7 +11,7 @@ class VerificationHandler < CensusHandler
 
           validates :document_file1, presence: true
           validates :document_file2, presence: true, if: :require_document_file2?
-          validates :tos_agreement, allow_nil: false, acceptance: true
+          validates :tos_agreement, allow_nil: true, acceptance: true
```

Comment:
Why this? What are the effects of the `allow_nil` here?

Reply:
I explained this in the PR description.

Reply:
To further elaborate, without this, the document verification step will fail and give no clue about it. The alternative could be to raise if there's no terms and conditions page for verifications, instead of conditionally render the checbox?

Reply:
To further elaborate, without this, the document verification step will fail and give no clue about it. The alternative could be to raise if there's no terms and conditions page for verifications, instead of conditionally render the checbox?

Reply:
:see_no_evil: sorry, it's ok this way

Reply:
No problem 😉